app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,19 @@
+class ReservationCalendar < SimpleDelegator
+
+  def initialize(reservation)
+    super(reservation)
+  end
+
+  def as_ical
+    cal = Icalendar::Calendar.new
+    cal.event do |e|
+      e.dtstart = Icalendar::Values::DateTime.new(reserve_start_at)
```

Comment:
My assumption is that we don't need to worry about dates being in different timezones, so I don't need to worry about conversion?